frontend/packages/console-dynamic-plugin-sdk/src/lib-host-app.ts

```diff
@@ -1 +1,7 @@
+/**
+ * @file host-app entrypoint for the dynamic plugin SDK. Provides APIs, Component, and utilities for host applications i.e reducers, actions, etc
+ *
+ * Published to NPM
```

Comment:
This package isn't actually published to npmjs :) we can remove it in a follow-up PR.

Reply:
Will be removed in https://github.com/openshift/console/pull/11043/files#diff-022d760da705a74cf2f3dc3b0adf914fac1387ff8a6b4363821f6fb9e8e8cbf2L1

Reply:
Per discussion in slack, I will undo my changes to these files as it is removed in: https://github.com/openshift/console/pull/11043